.github/SUPPORT.md

```diff
@@ -0,0 +1,49 @@
+# Getting help with lintr
+
+Thanks for using lintr. Before filing an issue, there are a few places
+to explore and pieces to put together to make the process as smooth as possible.
+
+Start by making a minimal **repr**oducible **ex**ample using the 
+[reprex](http://reprex.tidyverse.org/) package. If you haven't heard of or used 
+reprex before, you're in for a treat! Seriously, reprex will make all of your 
+R-question-asking endeavors easier (which is a pretty insane ROI for the five to 
+ten minutes it'll take you to learn what it's all about). For additional reprex
+pointers, check out the [Get help!](https://www.tidyverse.org/help/) section of
+the tidyverse site.
+
+The most useful function to create reprexes for `{lintr}` issues is `lint()`.
+You can include code that doesn't lint as expected with the linter in question.
+For example,
+
+```r
+lintr::lint(
+  text = "x = 1",
+  linters = assignment_linter()
```

Comment:
lintr::

Reply:
I added `library()` call instead. I think that's the most user-friendly option here.

Reply:
agree